README.md

```diff
@@ -8,11 +8,11 @@
 [![Build Status](https://circleci.com/gh/iodide-project/pyodide.png)](https://circleci.com/gh/iodide-project/pyodide)
 [![Documentation Status](https://readthedocs.org/projects/pyodide/badge/?version=latest)](https://pyodide.readthedocs.io/?badge=latest)
 
-The Python scientific stack, compiled to WebAssembly.
+Python and the scientific stack, compiled to WebAssembly.
```

Comment:
What about

> Python, including the scientific stack, compiled to WebAssembly.

? Or 

> Python in WebAssembly - And no compromises!

okay latter one is a little joke maybe. For me, your original draft is also just fine.